backend/Controllers/UserController.cs

```diff
@@ -21,4 +23,59 @@ public async Task<IActionResult> GetByUserName([FromRoute] string userName)
         }
         return Ok(new UserResponse { Id = matchingUser.Id, UserName = matchingUser.UserName!, });
     }
+
+    [HttpGet("GetUserId/{userId}")]
+    public async Task<User?> GetByUserId([FromRoute] string userId)
+    {
+        var matchingUser = await _userManager.FindByIdAsync(userId);
+        if (matchingUser == null)
+        {
+            return null;
+        }
+        return matchingUser;
+    }
+
```

Comment:
I'm not sure this is meant to be in this PR - looks like it might be part of another ticket?